src/query/execute_statement.c

```diff
@@ -5700,17 +5700,24 @@ check_trigger (DB_TRIGGER_EVENT event, PT_DO_FUNC * do_func, PARSER_CONTEXT * pa
 	  if (node->info.spec.flag & PT_SPEC_FLAG_DELETE)
 	    {
 	      flat = node->info.spec.flat_entity_list;
-	      class_ = (flat ? flat->info.name.db_object : NULL);
-	      if (class_ == NULL)
+	      if (node->info.spec.remote_server_name)
```

Comment:
How about 'IS_DBLINK_DML' for readability?

Reply:
I'd like to keep the logic to check remote server name easily.